calicoctl/commands/apply.go

```diff
@@ -57,6 +57,7 @@ Description:
     * bgpPeer
     * felixConfiguration
     * globalNetworkPolicy
+    * globalNetworkSet
```

Comment:
ick, I think these are all wrong :/

All the resource types start uppercase now - `BGPPeer`, `GlobalNetworkPolicy`, etc...

happy to ignore for now and we can fix in another PR.

Reply:
Raised #1800